Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Replacement of TOG Looting Events

### Changelog:
- Replaced random TOG looting events with random event that spawns artifacts from mod after a successful raid. Gamerule to regulate how often this event is fired included.


### Developer changelog:
- Removed `give_new_artifact_random_name_and_origin_effect` from artifact spawn effects.


### How to test:

- Event
  - `event test.1` in order to increase event fire chance by 9999. Then siege down some holdings in raid.
- Gamerule
  - Run a game in debug mode for a few years in observer mode, open `Crusader Kings II\logs\game.log` and check how much generic artifacts were spawned on_loot_settlement.
- Balance
  - Tell me if you don't like prestige values when you refuse to take an artifact.
- Proofread
  - [this](https://docs.google.com/spreadsheets/d/1FVVfKRCmTzPfiqoKvcfllqOhR613hpCzm2XJ8Gk1moU/edit#gid=1189421545&range=1:10) spreadsheet and newly added localization lines on github
